Lombiq.Privacy.Tests.UI/Extensions/TestCaseUITestContextExtensions.cs

```diff
@@ -71,14 +71,14 @@ public static async Task TestConsentBannerContentAsync(this UITestContext contex
 
     public static async Task TestConsentBannerAcceptButtonAsync(this UITestContext context)
     {
-        var privacyConsentAcceptButton = By.Id(ElementSelectors.PrivacyConsentAcceptButtonId);
-
         await context.GoToHomePageAsync();
 
-        await context.ClickReliablyOnAsync(privacyConsentAcceptButton);
+        var privacyConsentAcceptButton = By.Id(ElementSelectors.PrivacyConsentAcceptButtonId);
 
         try
         {
+            await context.ClickReliablyOnAsync(privacyConsentAcceptButton);
```

Comment:
Do we really want to move this inside the `try { ... }` block?

As I checked the logs, the `StaleElementReferenceException` was thrown by a `ElementRetrievalUITestContextExtensions.Missing` call, most possibly here:

https://github.com/Lombiq/Orchard-Privacy/blob/aa7d2d693d3382d6352637db5c11f2bcf8ff1ad1/Lombiq.Privacy.Tests.UI/Extensions/TestCaseUITestContextExtensions.cs#L93

![image](https://user-images.githubusercontent.com/15349881/211603509-b0049b32-cda1-4199-a047-ae7d05defcda.png)

Reply:
Yes, indeed, I missed that it's coming from a second `Missing()` (since another one was already guarded against the same issue), thank you.